### PR TITLE
DVX-4446: Adds VHost redirections to a vhost input

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

File: mcc/trek/trek/trek.go [hunk 1]
```diff
@@ -9,7 +9,8 @@ import (
 	"regexp"
 )
 
-// RFC1123HostnameRegExp : Hostname Regular expression according with RFC1123HostnameRegExp.
+// RFC1123HostnameRegExp : Hostname Regular expression according with
+// RFC1123HostnameRegExp.
 const RFC1123HostnameRegExp = "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|\\*)\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]*[A-Za-z0-9])$"
```
Comment (ifosch) [inline mcc/trek/trek/trek.go:14]: @miguelbernadi I would consider some other way to represent these strings that fits in 80 cols, but that might make code less clear,... do you think it's worht the effort?

File: mcc/trek/trek/trek_test.go [hunk 2]
```diff
@@ -64,4 +65,26 @@ func TestAdd(t *testing.T) {
 	if err == nil {
 		t.Errorf("Both original and final should be present")
 	}
+	vhosts := "server {\n" +
```
Comment (ifosch) [inline mcc/trek/trek/trek_test.go:68]: @mbernabe Maybe we could have a helper to build this, or using a template... Again, do you think this is valuable effort, now? I think not, while we don't need this anywhere else.

Comment (ifosch) [reply mcc/trek/trek/trek_test.go:68]: @miguelbernadi

**Comment by ifosch** [comment]
[B] for @miguelbernadi review

File: mcc/trek/trek/trek.go (hunk 1 above)
Comment (miguelbernadi) [reply mcc/trek/trek/trek.go:14]: I don't think it's worth it, at least not now.

File: mcc/trek/trek/trek_test.go (hunk 2 above)
Comment (miguelbernadi) [reply mcc/trek/trek/trek_test.go:68]: I don't think it's valuable for this PR, but we should use templates eventually. We'll need this kind of functionality again in the future so we can change it then.

**Review by miguelbernadi — APPROVED** [review]
+1

**Comment by miguelbernadi** [comment]
+1, in case the review comment isn't seen by Jarvis.

**Comment by christofdamian** [comment]
+1

**Pull request merged**